shell/browser/api/electron_api_web_frame_main.cc

```diff
@@ -185,7 +185,8 @@ void WebFrameMain::Send(v8::Isolate* isolate,
     return;
 
   GetRendererApi()->Message(internal, channel, std::move(message),
-                            0 /* sender_id */);
+                            0 /* sender_id */,
+                            false /* sender_is_main_frame */);
```

Comment:
Given `WebFrameMain` can be created for main frame, should this also check ?
```suggestion
                            !Parent() /* sender_is_main_frame */);
```

Reply:
The new `senderIsMainFrame` property is only relevant for messages with non-zero `senderId`